base/acme/src/main/java/org/dogtagpki/acme/scheduler/ACMETask.java

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright Red Hat, Inc.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+package org.dogtagpki.acme.scheduler;
+
+/**
+ * @author Endi S. Dewata
+ */
+public class ACMETask {
```

Comment:
Should this class be declared `abstract`?

Reply:
Updated.